src/extractors.rs

```diff
@@ -50,8 +53,8 @@ where
     Store: UserStore<Role, User = User>,
     Role: PartialEq + Clone + Send + Sync + 'static,
 {
-    fn get_session_auth_id(&self, password_hash: &str) -> String {
-        let tag = hmac::sign(&self.key, password_hash.as_bytes());
+    fn get_session_auth_id(&self, password_hash: &[u8]) -> String {
```

Comment:
Does this change need to be behind a feature flag as well?

Reply:
I don't think so - this is not a public function.